crates/fuel-gas-price-algorithm/src/lib.rs

```diff
@@ -194,7 +194,7 @@ pub struct RecordedBlock {
     pub block_cost: u64,
 }
 
-#[derive(Debug, Clone, PartialEq)]
+#[derive(serde::Serialize, serde::Deserialize, Debug, Clone, PartialEq)]
```

Comment:
K. Just created a `V1NoDaMetadata` type for now. It doesn't have any concept of DA and when used to generate an Updater just zeroes out all the DA fields.

I just realized that we still have the problem of the `unrecorded blocks` on the updater itself. It will grow forever currently.  If we ever restart the updater it will be empty.

Instead we could just create an updater that doesn't have any of that. That's probably best for now.

Reply:
Okay... I just created a `V0` which removes all the DA stuff. We're using that instead so we don't even need to worry about all the overhead at all.